content/en/news/releases/1.18.x/announcing-1.18/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Announcing Istio 1.18.0
+linktitle: 1.18.0
+subtitle: Major Release
+description: Istio 1.18 Release Announcement.
+publishdate: 2023-06-02
```

Comment:
Still old date, noticed in list of news items.

Reply:
I'll update the dates across all the files when we have locked in a date later today